src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -194,6 +211,10 @@ import ActionButton from '../ActionButton/ActionButton'
 import AppNavigationIconCollapsible from './AppNavigationIconCollapsible'
 import isMobile from '../../mixins/isMobile'
 import InputConfirmCancel from './InputConfirmCancel'
+import { t } from '../../l10n'
+
+import Pencil from 'vue-material-design-icons/Pencil.vue'
+import Undo from 'vue-material-design-icons/Undo.vue'
```

Comment:
We should either add extentions to all import paths or keep them off :)

Reply:
True. I removed the filetype extension. And took the liberty to remove it in two other files as well here.